### ISSUE TITLE
Compiled executables don't work

### ISSUE THREAD

**Issue by zsombro** [asos-craigmorten/opine#107]
# Issue

Setup:

- Deno Version: 1.8.0
- v8 Version: 9.0.257.3
- Typescript Version: 4.2.2
- Opine Version: 1.1.0

## Details

> I have replicated the Eta templating example for Opine and while the example itself worked, when I try to compile this program to a Windows executable, I get the following error upon running it:

```
error: ReferenceError: to is not defined
    at Function.app1.on (file://$deno$/bundle.js:30032:42)
    at Function.defaultConfiguration (file://$deno$/bundle.js:28769:10)
    at Function.init (file://$deno$/bundle.js:28756:10)
    at opine (file://$deno$/bundle.js:30051:10)
    at file://$deno$/bundle.js:32779:14
```

I have tried compiling executables before, but they only used standard libraries. I assumed this might be a problem with opine, as the callstack in that error message refers to it. I don't know if there are any command line arguments that could help in producing a more helpful error log

**Comment by cmorten** [comment]
Heya @zsombro 👋 

I've just released [`1.2.0`](https://github.com/asos-craigmorten/opine/releases/tag/1.2.0) - please can you confirm whether you are still facing issues?

I am able to compile and run the `eta` example in this codebase with this latest version - see video.

https://user-images.githubusercontent.com/11313985/110216322-6f2fb300-7ea6-11eb-949a-8719b99e2abe.mov

If you're still facing issues, please could you share a minimal code example ( code block / gist ) so I can use that to reproduce and work on a fix? 😄

**Comment by zsombro** [comment]
Hey @cmorten thank you for the quick response!

I have tried the new version and it works fine! Awesome!
